react/Table/Readme.md

```diff
@@ -118,7 +124,7 @@ const onSortChange = ({ order, orderBy }) => {
         rows={rows}
         columns={columns}
         groups={variant.grouped ? makeGroups : undefined}
-        defaultOrder={columns[0].id}
+        defaultOrder={{ attribute: 'calories', order: 'desc' }}
```

Comment:
`desc` is a keyword, `calories` is not so I think `columns[0].id` is more relevant, specially if the column construction changes